### PR TITLE
scripts: add support for paginated responses

### PR DESCRIPTION
**Pull request by lheckemann**
Thanks hub, for making it easy!

### CODE REVIEW

File: nominations.jq [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
-   | add
-end
\ No newline at end of file
+map(.
+  | "(?<str>shep|nominat)" as $regex
+  | select(.body | match($regex; "i"))
+  | { body: .body | gsub($regex; "\u001b[31;1m" + .str + "\u001b[m"; "i"), author: .user.login }
+  | .author + "\n---\n" + .body + "\n\n"
+)
+| add
```
Comment (SuperSandro2000) [inline nominations.jq:7]: Missing final newline.

File: nominations.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p jq curl
+#!nix-shell -i bash -p jq gitAndTools.hub
```
Comment (SuperSandro2000) [inline nominations.sh:2]:
```suggestion
#!nix-shell -i bash -p jq hub
```

deprecated alias

File: rfc-report.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p jq curl
+#!nix-shell -i bash -p jq gitAndTools.hub
```
Comment (SuperSandro2000) [inline rfc-report.sh:2]:
```suggestion
#!nix-shell -i bash -p jq hub
```

**Comment by Mic92** [comment]
Applied to master.

**Pull request closed without merging**